plugin/parser/differ/mysql/differ.go

```diff
@@ -105,5 +106,35 @@ func buildTableMap(nodes []ast.StmtNode) map[string]*ast.CreateTableStmt {
 
 // isTwoColumnsSame returns true if the two columns are the same.
 func isTwoColumnsSame(old, new *ast.ColumnDef) bool {
-	return old.Tp.String() == new.Tp.String()
+	if old.Tp.String() != new.Tp.String() {
+		return false
+	}
+	oldNormalizeOptions := normalizeOptions(old.Options)
+	newNormalizeOptions := normalizeOptions(new.Options)
+	if len(oldNormalizeOptions) != len(newNormalizeOptions) {
+		return false
+	}
+	for idx, oldOption := range oldNormalizeOptions {
+		newOption := newNormalizeOptions[idx]
+		if oldOption.Tp != newOption.Tp {
+			return false
+		}
+	}
+	return true
+}
+
+// normalizeOptions normalizes the column options.
+// It skips the NULL option and order the options by OptionType.
+func normalizeOptions(options []*ast.ColumnOption) []*ast.ColumnOption {
```

Comment:
nit: normalizeColumnOptions in case we have other stuff to normalize later.

Reply:
Done.